Q/qwtw/build_tarballs.jl

```diff
@@ -52,4 +52,4 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.6", preferred_gcc_version = v"9.1.0")
+               julia_compat="1.8", preferred_gcc_version = v"9.1.0")
```

Comment:
Any specific reason for this change?

Reply:
no any specific reason for this. The only change that might be important here is a move from C++14 into C++20..

Reply:
If there is no reason to change the julia_compat, then maybe just don't change it?

Reply:
```suggestion
               julia_compat="1.6", preferred_gcc_version = v"9.1.0")
```